packages/new-polymath-issuer/src/pages/DividendsWizard/Presenter.tsx

```diff
@@ -50,17 +52,47 @@ export interface Props {
 
 export interface State {
   excludedWallets: null | ExclusionEntry[];
+  dividendAmount: BigNumber;
+  positiveWithholdingAmount: number;
 }
 
 export class Presenter extends Component<Props> {
-  public state = {
+  public state: State = {
```

Comment:
The state type should go on `Component<Props, State>`

Reply:
Yeah, but it's still needed there, otherwise I get type errors when accessing some properties that are initialized as `null`

Reply:
Should still be added to the `Component` generic as a second argument though right?

Reply:
Yeah, I did that